architecture/sync.md

```diff
@@ -82,15 +82,15 @@ Changes are first placed in the `pending` collection before they are applied to
         - `last_modified` (set to the relevant server timestamp, or `0` if not known)
         - `deleted` (set to `true`)
 
-* `conflicts`: [**integer**] - the record in the `pending` collection this record conflicts with (missing or `0` if it does not conflict).
+* `conflicts`: [**integer**] - the index of a record in the `pending` collection with which this record conflicts (absent or `0` if it does not conflict).
 
-Whenever a change is to be tracked (whether that change comes from remote storage or a local modification), the `pending` collection is queried for an existing record (by source, collection, and id) before a record is inserted.  If there is an existing `pending `record, it is updated with the latest; otherwise a new record is inserted.
+Whenever a change is to be tracked (whether that change comes from remote storage or a local modification), the `pending` collection is queried for an existing record (by source, collection, and id) before a record is inserted.  If there is an existing `pending` record, it is updated with the latest changes; otherwise a new record is inserted.
 
-When an item or keystore is changed locally, a record is inserted/updated into the `pending` collection rather than directly applying to the `items` or `keystores`.  When remote changes are fetched, a record for each is inserted/updated into the `pending` collection.
+When an item or keystore is changed locally, a record is inserted/updated into the `pending` collection rather than directly applying to the `items` or `keystores` collections.  When remote changes are fetched, a record for each is inserted/updated into the `pending` collection.
 
 ### Markers
 
-In addition to tracking the actual changes, the markers (i.e., [server timestamps](#server-timestamps)) are also tracked locally. The marker for each collection is stored on the device in IndexedDB via the `markers` collection:
+In addition to tracking the actual changes, the markers (_i.e._, [server timestamps](#server-timestamps)) are also tracked locally. The marker for each collection is stored on the device in IndexedDB via the `markers` collection:
```

Comment:
Should this be "i.e." or "e.g." (to match change above on L28)?

Reply:
this really is a case of "that is" (i.e.) rather than "for instance" (e.g.).